docker-compose.yml

```diff
@@ -28,16 +28,17 @@ services:
     container_name: ops-migration
     command: >
       bash -c "
-      python ./opre_ops/manage.py custom-flush --allow-cascade --no-input &&
-      python ./opre_ops/manage.py migrate &&
-      python ./opre_ops/manage.py loaddata ./opre_ops/ops_site/fixtures/fake_data.json
+      python ./ops_api/manage.py custom-flush --allow-cascade --no-input &&
+      python ./ops_api/manage.py makemigrations &&
```

Comment:
Just checking that you actually want this here.  This will potentially make a new migration file and will probably show up in your local copy of the repository (since we have a Docker volume sharing).

Reply:
Since the docker compose deploy is only used for development I think it's useful to have here - it ensures the test data is fresh and any migrations are identified at dev time.  I could be convinced otherwise but I think it's a nice to have.